s4e-web/src/app/views/settings/manage-institutions/institution-list/institution-list.component.html

```diff
@@ -9,7 +9,9 @@
             queryParamsHandling="merge" i18n>Dodaj nową instytucję</button>
   </ng-container>
   <ng-container description></ng-container>
-  <ng-container caption i18n>Zarządzanie instytucjami</ng-container>
+  <h1 caption i18>
+    <ng-container i18n>Lista instytucji</ng-container>
```

Comment:
`<ng-container i18n>Lista instytucji</ng-container>` can be just simple `Lista instytucji`, because `<ng-container></ng-container>` will be not visible `DOM element` after compilation (only it's content `Lista instytucji`)